### PR TITLE
Board listeners

### PR DESCRIPTION
**Pull request by slayerhintze**
Added board classes event listeners

### CODE REVIEW

File: index.html
Comment (jbeBan) [inline index.html:1]: Thank you for catching that extra head tag.

File: js/index.js [hunk 1]
```diff
@@ -1,3 +1,272 @@
+// Function that handles adding event listeners to the chess board
+function initBoardEvListener() {
+    // First column. 'a' column
+    const a1Board = document.getElementById("a1");
+    a1Board.addEventListener("click", function(event) {
+    })
```
Comment (jbeBan) [inline js/index.js]: Each call to `addEventListener()` is missing a semicolon at the end of the statement. Please add a semicolon at the end of each call to `addEventListener()`.

**Review by jbeBan — CHANGES_REQUESTED** [review]

**Review by jbeBan — APPROVED** [review]
Perfect! Thanks for making those changes.

**Review by Jason-mcm — APPROVED** [review]
Looks great!

**Pull request merged**